drift_dev/lib/src/analysis/resolver/file_analysis.dart

```diff
@@ -73,6 +73,14 @@ class FileAnalyzer {
               .whereType<DriftElement>()
               .followedBy(availableByDefault)
               .transitiveClosureUnderReferences()
+              .where((e){
+                if (element is DriftDatabase) {
+                  if ((e is DriftTable) && e.baseDartName.startsWith('_')) {
+                    return e.id.libraryUri == element.id.libraryUri;
+                  }
```

Comment:
Add a comment explaining the intention of only including public elements of other libraries, allowing private table classes to be defined without breaking the build.

Reply:
Absolutely, done.